app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/payments/PaymentsViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package com.hedvig.android.feature.payments.payments
+
+import com.hedvig.android.core.demomode.Provider
+import com.hedvig.android.feature.payments.overview.data.GetPaymentOverviewDataUseCase
+import com.hedvig.android.molecule.android.MoleculeViewModel
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.WhileSubscribed
+
+internal class PaymentsViewModel(
+  getPaymentOverviewDataUseCase: Provider<GetPaymentOverviewDataUseCase>,
+) : MoleculeViewModel<PaymentsEvent, PaymentsUiState>(
+    PaymentsUiState.Loading,
+    PaymentsPresenter(
+      getPaymentOverviewDataUseCase = getPaymentOverviewDataUseCase,
+    ),
+    SharingStarted.WhileSubscribed(0.seconds),
```

Comment:
Could you explain a bit this 0.seconds here?

Reply:
I can definitely explain it by saying that this was a mistake 😂
Thanks for noticing!